### PR TITLE
HRM02: Commit - MVVM base structure

### LINKED ISSUE #2: HRM02-Create clean architecture structure

### PR DESCRIPTION
**Pull request by akshaya86**
-  Created module structure

### CODE REVIEW

File: .idea/gradle.xml [hunk 1]
```diff
@@ -10,6 +10,8 @@
           <set>
             <option value="$PROJECT_DIR$" />
             <option value="$PROJECT_DIR$/app" />
+            <option value="$PROJECT_DIR$/data" />
+            <option value="$PROJECT_DIR$/domain" />
```
Comment (EntyR) [inline .idea/gradle.xml]: please, add all .idea folder into gitignore

File: data/build.gradle [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    }
+}
+
+dependencies {
+
+    implementation 'androidx.core:core-ktx:1.7.0'
+    implementation 'com.google.android.gms:play-services-wearable:17.1.0'
+    implementation 'androidx.percentlayout:percentlayout:1.0.0'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation 'androidx.recyclerview:recyclerview:1.2.1'
```
Comment (EntyR) [inline data/build.gradle]: I'm not sure all of this dependencies needed here

**Review by EntyR — CHANGES_REQUESTED** [review]

**Comment by EntyR** [comment]
Please delete all unused resource and dependencies from data and domain module

File: data/build.gradle (hunk 2 above)
Comment (akshaya86) [reply data/build.gradle]: Resolved

File: .idea/gradle.xml (hunk 1 above)
Comment (akshaya86) [reply .idea/gradle.xml]: added folder to gitingore

**Comment by akshaya86** [comment]
@EntyR Please revisit the PR

**Review by EntyR — CHANGES_REQUESTED** [review]
Please also delete ic_launcher from domain and data no need for them especially in domain module

**Comment by akshaya86** [comment]
@EntyR  Please revisit this PR

**Review by EntyR — APPROVED** [review]

**Comment by EntyR** [comment]
Very nice), I'm not collaborator, so can merge it yourself

**Comment by EntyR** [comment]
image
Also would be great if add app project name in other module, but up to you

**Comment by akshaya86** [comment]
@EntyR  Thank You

**Pull request merged**